src/pilotdatasync-xp11.cpp

```diff
@@ -32,12 +32,14 @@ DllMain(HANDLE hModule, DWORD ul_reason_for_call, LPVOID lpReserved) {
 static XPLMWindowID g_window;
 
 // DataRef Identifiers
-static XPLMDataRef elevationMslRef;
-static XPLMDataRef elevationAglRef;
-static XPLMDataRef airspeedRef;
-static XPLMDataRef verticalVelocityRef;
-static XPLMDataRef headingPilotRef;
+static XPLMDataRef elevationFlightmodelRef;
+static XPLMDataRef elevationPilotRef;
+static XPLMDataRef airspeedFlightmodelRef;
+static XPLMDataRef airspeedPilotRef;
```

Comment:
possibly better specify which data refs are based on the flightmodel and which are based on what the game things